src/implementations/vanilla/components/social-media-follow/social-media-follow.scss

```diff
@@ -0,0 +1,106 @@
+/**
+* Social media follow component
+* @define social-media-follow
+*/
+
+@use "sass:map";
+@use '@ecl/theme-dev/maps/color';
+@use '@ecl/theme-dev/maps/spacing';
+@use '@ecl/theme-dev/maps/typography';
+@use '@ecl/vanilla-layout-grid/mixins/breakpoints';
+
+.ecl-social-media-follow {
+  background-color: map.get(color.$color, 'grey-5');
```

Comment:
There are 2 private variables in the original code

```scss
@mixin ecl-social-media-follow(
  $background: $ecl-color-grey-5,
  $description-color: $ecl-color-black-100
) {}
```

In the other components they are hoisted and renamed to have underscores to be really private
Example: https://github.com/ec-europa/europa-component-library/blob/next-v3/src/implementations/vanilla/components/button/button.scss#L13-L16

I personally don't mind reworking files to not include these private variables. Though for me personally it's easier to take them and rename rather than reworking the whole file to not include them. With 2 such props it's ok, but with more it becomes harder ...

Reply:
Ok thank you for this remark, taking note of the other components 👍 
Here as the variables were only used once, I preferred to remove them.